lib/umwelt/files.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Umwelt.Files do
+  @moduledoc "Finds elixir files recursive from dir"
+
+  def list_from_root(project),
+    do: files_in("lib/#{project}")
+
+  def files_in(path) do
+    {:ok, files} = File.ls(path)
+
+    files
+    |> Enum.map(&extract("#{path}/#{&1}"))
+    |> List.flatten()
```

Comment:
```suggestion
    |> Enum.flat_map(&extract(Path.join(path, &1)))
```

`List.flatten/1` is your enemy. When it was introduced I fought José for bringing bad habits to _Elixir_ and he said “easier for newcomers, blah-blah.”